package.json

```diff
@@ -8,14 +8,22 @@
     "lib"
   ],
   "devDependencies": {
-    "suitcss-components-test": "*",
-    "suitcss-preprocessor": "~0.3.0"
+    "autoprefixer": "^6.0.3",
+    "postcss": "^5.0.8",
+    "postcss-bem-linter": "^1.1.0",
+    "postcss-calc": "^5.0.0",
+    "postcss-cli": "^2.1.0",
+    "postcss-custom-media": "^5.0.0",
+    "postcss-custom-properties": "^5.0.0",
+    "postcss-import": "^7.0.0",
```

Comment:
We're using this one, as it's imported into `index.css`. The tests make use of it too.